### PR TITLE
feat: add GetDirections component

### PR DESCRIPTION
**Pull request by colton-demetriou**
tested manually with https://dev.yext.com/s/1000146856/yextsites/63179/editor#templateId=location&layoutId=8747046&view=EDIT&entityId=8694637

### CODE REVIEW

File: src/components/GetDirections.tsx
Comment (jwartofsky-yext) [inline src/components/GetDirections.tsx:1]: Ben fixed the issue with themes not working for components in visual-editor, so we should put all the components there now.

File: src/templates/location.tsx [hunk 1]
```diff
@@ -49,6 +49,7 @@ export const config = {
       "services",
       "c_deliveryPromo",
       "c_testImage",
+      "yextDisplayCoordinate",
```
Comment (jwartofsky-yext) [inline src/templates/location.tsx:52]: this should be above the c_ fields I think

File: src/components/GetDirections.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  size,
+  alignment,
+  padding,
+  coordinate: coordinateField,
+  getDirectionsProvider,
+}: GetDirectionsProps) => {
+  const document = useDocument<LocationStream>();
+  let coordinate = resolveYextEntityField<Coordinate>(document, coordinateField);
+  if (!coordinate) {
+   console.warn("yextDisplayCoordinate is not present in the stream");
```
Comment (jwartofsky-yext) [inline src/components/GetDirections.tsx:97]: in visual-editor, we may be able to use the xYextDebug methods here

Comment (colton-demetriou) [reply src/components/GetDirections.tsx:97]: will update later if so

**Pull request closed without merging**